src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -59,12 +60,45 @@
     private final UserService userService;
     private final DataAccessRequestService darService;
 
+    private final String defaultDataSetSampleFileName = "DataSetSample.tsv";
+    private final String defaultDataSetSampleContent = "Dataset Name\tData Type\tSpecies\tPhenotype/Indication\t# of participants\tDescription\tdbGAP\tData Depositor\tPrincipal Investigator(PI)\tSample Collection ID\tConsent ID"
+            + "\n(Bucienne Monco) - Muc-1 Kidney Disease\tDNA, whole genome\thuman\tmuc-1, kidney disease\t31\tmuc-1 patients that developed cancer , 5 weeks after treatment\thttp://....\tJohn Doe\tMark Smith\tSC-20658\t1";
+
+    private String dataSetSampleFileName;
+    private String dataSetSampleContent;
+
+    String getDataSetSampleFileName() {
+        return dataSetSampleFileName;
+    }
+
+    void setDataSetSampleFileName(String fileName) {
+        dataSetSampleFileName = fileName;
+    }
```

Comment:
"DataSetSample.tsv" could be a private constant in this file so you don't have to copy the string here and on line 63.